### PR TITLE
ISPN-14528 Enable configuration conversion for templates

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-14528

### CODE REVIEW

**Comment by tristantarrant** [comment]
Failures are unrelated

File: core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java [hunk 1]
```diff
@@ -1162,21 +1171,29 @@ protected void parseScatteredCache(ConfigurationReader reader, ConfigurationBuil
       }
    }
 
-   private ConfigurationBuilder getConfigurationBuilder(ConfigurationBuilderHolder holder, String name, boolean template, String baseConfigurationName) {
+   private ConfigurationBuilder getConfigurationBuilder(ConfigurationReader reader, ConfigurationBuilderHolder holder, String name, boolean template, String baseConfigurationName) {
       if (holder.getNamedConfigurationBuilders().containsKey(name)) {
          throw CONFIG.duplicateCacheName(name);
       }
+      boolean ignoreMissingTemplates = reader.getProperty(IGNORE_MISSING_TEMPLATES) != null;
```
Comment (jabolina) [inline core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java:1178]: Should we check the value as well? Setting the property to false would evaluate to true here.

Comment (tristantarrant) [reply core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java:1178]: It's an undocumented property used only internally

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Merged into main, thanks @tristantarrant!

**Pull request merged**